optaweb-vehicle-routing-distribution/src/main/assembly/assembly-optaweb-vehicle-routing.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+
+  <id>assembly-optaweb-vehicle-routing</id>
+  <formats>
+    <format>dir</format>
+    <format>zip</format>
+  </formats>
+
+  <includeBaseDirectory>true</includeBaseDirectory>
+
+  <fileSets>
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <includes>
+        <include>../LICENSE-Apache-2.0.txt</include>
+      </includes>
+      <outputDirectory/>
+    </fileSet>
+    <!-- Examples -->
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <directory>..</directory>
+      <outputDirectory>sources</outputDirectory>
+      <excludes>
+        <exclude>**/target/**</exclude>
+        <exclude>**/local/**</exclude>
+        <exclude>.*/**</exclude>
+        <exclude>nbproject/**</exclude>
+        <exclude>*.ipr</exclude>
+        <exclude>*.iws</exclude>
+        <exclude>**/*.iml</exclude>
+        <exclude>.git/**</exclude>
+        <exclude>**/node_modules/**</exclude>
+        <exclude>optaweb-vehicle-routing-distribution/**</exclude>
+        <exclude>**/.scannerwork/**</exclude>
```

Comment:
Please also exclude `coverage`, `node` and `build` directories. These all contain results of the build or local copy of build tools (the node dir). For illustration, content of the distribution's frontend module now:
```
$ du -hs * | sort -h
4.0K	package.json
4.0K	README.adoc
4.0K	tsconfig.json
8.0K	pom.xml
40K	public
552K	src
684K	package-lock.json
1.1M	coverage
14M	build
34M	node
```